src/Event.cs

```diff
@@ -62,6 +62,10 @@ public SubscriptionBuilder<EventContext> OnError(Action<EventContext> callback)
         }
 
         public SubscriptionHandle<EventContext> Subscribe(params string[] querySqls) => new(conn, Applied, Error, querySqls);
+
+        public void SubscribeToAllTables() {
+            new SubscriptionHandle<EventContext>(conn, Applied, Error, "SELECT * FROM *");
```

Comment:
Oops, sorry, it's the `Subscribe` function that has variadic signature, not the constructor, so this should actually be just

```suggestion
            Subscribe("SELECT * FROM *");
```